src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A deadline task that extends from Task.
+ *
+ * @author Erwin Quek
+ * @version CS2103 AY21/22 Sem 1
+ */
+public class Deadline extends Task{
+    private String dateTime;
+    private final LocalDateTime dt;
+
+    /**
+     * Constructor for Deadline.
+     * @param description Takes in a description of Deadline task
```

Comment:
A description doesn't take in a description

Reply:
ok noted! I will make the necessary changes. Thank you 😄